packages/koa-shopify-auth/client/enable-cookies.js

```diff
@@ -0,0 +1,45 @@
+(function() {
+  function setCookieAndRedirect() {
+    document.cookie = 'shopify.granted_storage_access=true';
```

Comment:
I just thought of this question now, but should have thought of this earlier: Should the cookies we're setting for both the ITP 2.0 and ITP 2.1 workflows be unique to the app being authenticated (e.g., by adding the app ID to the cookie name)? This question also applies to the `shopify_app` gem

cc @ragalie

Reply:
Nevermind, I was having a brain fart moment. Talked it through with Mike and this shouldn't be an issue. Sorry!